annet/annlib/jsontools.py

```diff
@@ -87,3 +87,27 @@ def apply_patch(content: Optional[bytes], patch_bytes: bytes) -> bytes:
 
     new_contents = format_json(new_doc, stable=True).encode()
     return new_contents
+
+
+def _apply_path(doc: Dict[str, Any], path: List[str]):
```

Comment:
I would put it inside `apply_acl_filters` itself, do not see any usage outside of it

Reply:
You mean that it should be defined inside `apply_acl_filters`? This is a recursive function. It was separated so that it can call itself.

Reply:
Yes, I meant the definition.

PS: to honest I do not see why it has to be recursive, all we do in it is iterating over the list